File: WPA.backend/WPA.backend/Entities/Guest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WPA.backend.Entities
{
    public class Guest
    {
        public int Id { get; set; }
        public int PlannerId { get; set; }
        public string Name { get; set; }
        public bool IsTravelling { get; set; }
        public string Adnotation { get; set; }
        public GuestStatus Status { get; set; }
        public GuestSide Side { get; set; }
    }
}

File: mobile/src/app/api/models/guest.ts
/* tslint:disable */
import { GuestSide } from './guest-side';
import { GuestStatus } from './guest-status';
export interface Guest {
  adnotation?: string;
  id: number;
  isTravelling: boolean;
  name?: string;
  plannerId: number;
  side: GuestSide;
  status: GuestStatus;
}
